spec.html

```diff
@@ -24906,7 +24953,7 @@ <h1>Object.prototype.toString ( )</h1>
           1. Return the string-concatenation of *"[object "*, _tag_, and *"]"*.
         </emu-alg>
         <emu-note>
-          <p>Historically, this function was occasionally used to access the String value of the [[Class]] internal slot that was used in previous editions of this specification as a nominal type tag for various built-in objects. The above definition of `toString` preserves compatibility for legacy code that uses `toString` as a test for those specific kinds of built-in objects. It does not provide a reliable type testing mechanism for other kinds of built-in or program defined objects. In addition, programs can use @@toStringTag in ways that will invalidate the reliability of such legacy type tests.</p>
+          <p>Historically, this function was occasionally used to access the String value of the [[Class]] internal slot that was used in previous editions of this specification as a nominal type tag for various built-in objects. The above definition of `toString` preserves compatibility for <emu-not-ref>legac</emu-not-ref>y code that uses `toString` as a test for those specific kinds of built-in objects. It does not provide a reliable type testing mechanism for other kinds of built-in or program defined objects. In addition, programs can use @@toStringTag in ways that will invalidate the reliability of such <emu-not-ref>legacy</emu-not-ref> type tests.</p>
```

Comment:
```suggestion
          <p>Historically, this function was occasionally used to access the String value of the [[Class]] internal slot that was used in previous editions of this specification as a nominal type tag for various built-in objects. The above definition of `toString` preserves compatibility for <emu-not-ref>legacy</emu-not-ref> code that uses `toString` as a test for those specific kinds of built-in objects. It does not provide a reliable type testing mechanism for other kinds of built-in or program defined objects. In addition, programs can use @@toStringTag in ways that will invalidate the reliability of such <emu-not-ref>legacy</emu-not-ref> type tests.</p>
```

assuming we keep the definition on lowercase "legacy" (see [discussion](https://github.com/tc39/ecma262/pull/2125/#discussion_r662639947)).